.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 115,
```

Comment:
115 eh?

Reply:
well, prettier for markdown does it wrong IMO. I uses the _start_ of a word, and not the end as the wrapping point. So, if you want to use the same line at 120, then you need to go back a bit.

Reply:
Pffffffff okay.